internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -1261,6 +1260,23 @@ func newImageServiceStatefulSet(ctx context.Context, log logrus.FieldLogger, asc
 			},
 		})
 
+		if asc.spec.OSImageCACertConfigMapRef != nil {
+			container.Env = append(container.Env, corev1.EnvVar{Name: "OS_IMAGE_DOWNLOAD_TRUSTED_CA_FILE", Value: "/etc/image-service/os-images-ca-bundle/extra-ca.pem"})
```

Comment:
Can you make this file name a constant since it's used here and in the validation?

Reply:
Also not sure why it needs to be called `extra` since it's the only cert the user is really aware of.
Maybe just `tls.crt` as that's the name used for our other serving certs.